index.html

```diff
@@ -740,7 +740,7 @@ <h5 class="heading settled" data-level="2.3.3.1" id="architecture"><span class="
      </ul>
    </ul>
    <p class="note" role="note"><span class="marker">Note:</span> For a comprehensive overview of Verifiable Credentials (VCs), refer to Ivan Herman’s <a href="https://www.w3.org/TR/vc-overview/">W3C Verifiable Credentials Overview</a>.</p>
-   <p>Another pillar of important element of this architecture are the <strong>Decentralized Identifiers (DIDs)</strong>, a new Uniform Resource Identifier (URI) type that allows entities and resources to be identified through various methods <a data-link-type="biblio" href="#biblio-did-core" title="Decentralized Identifiers (DIDs) v1.0">[did-core]</a>. These methods can rely on various technologies, including blockchains such as Bitcoin or Ethereum, the web, InterPlanetary File System (IPFS), and Domain Name System (DNS) [did-spec-registries]. All methods allow cryptographical control over the identifier. Still, they vary in how decentralized they are and the features they support (e.g., key rotation and revocation).</p>
+   <p>Another pillar of important element of this architecture are the <strong>Decentralized Identifiers (DIDs)</strong>, a new Uniform Resource Identifier (URI) type that allows entities and resources to be identified through various methods <a data-link-type="biblio" href="#biblio-did-core" title="Decentralized Identifiers (DIDs) v1.0">[did-core]</a>. These methods can rely on various technologies, including blockchains such as Bitcoin or Ethereum, the web, InterPlanetary File System (IPFS), and Domain Name System (DNS) @@missing reference: [did-spec-registries]. All methods allow cryptographical control over the identifier. Still, they vary in how decentralized they are and the features they support (e.g., key rotation and revocation).</p>
```

Comment:
it is the refernce for "These methods", often i put the reference at the end of the sentence. May I need to move upfront?

Reply:
Ah, I see! 
I don't know!
In any case, there is no link on the reference, wherever you put it.

Reply:
oh, i used the specref for that, i need to check the `.bs` file and fix

Reply:
checked, added `[[`